### PR TITLE
add convenience method to data provider (Organization) class

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java [hunk 1]
```diff
@@ -178,4 +178,12 @@ public abstract class DiseaseAnnotation extends Association {
 	@Transient
 	public abstract String getSubjectTaxonCurie();
 
+	public String getDataProviderString(){
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java:184]: @cmpich this needs to be @Transient also.

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java:184]: Plus can't this get done on the UI side? @adamgibs?

Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java:184]: I need this in the download logic of the API.

File: src/main/java/org/alliancegenome/curation_api/model/entities/Reference.java [hunk 2]
```diff
@@ -16,27 +13,56 @@
...
-	@JoinTable(indexes = { @Index(columnList = "Reference_curie"), @Index(columnList = "crossReferences_curie") })
+	@JsonView({View.FieldsOnly.class})
+	@JoinTable(indexes = {@Index(columnList = "Reference_curie"), @Index(columnList = "crossReferences_curie")})
 	@EqualsAndHashCode.Include
 	private List<CrossReference> crossReferences;
 
+
+	public static List<String> speciesPrefix;
+
+	static {
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/model/entities/Reference.java]: @cmpich can this be moved into a Enum... we need to have a enum package that can be used for this kind of thing.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**